src/locale/locales/fr/messages.po

```diff
@@ -6610,6 +6650,10 @@ msgstr "Pour envoyer des vidéos sur Bluesky, vous devez d’abord vérifier vot
 msgid "To whom would you like to send this report?"
 msgstr "À qui souhaitez-vous envoyer ce rapport ?"
 
+#: src/components/dialogs/nuxs/TenMillion/index.tsx:598
+msgid "Together, we're rebuilding the social internet. We're glad you're here."
+msgstr "Ensemble, nous reconstruisons l’internet social. Nous sommes heureux que vous soyez là."
```

Comment:
It's weird, but we say more often `web social` than `internet social` in French, so I think it's better to use that term. Also, `glad` is more `ravis` than `heureux`, but honestly `heureux` would be fine; to make sure that `here` is understood as `among us` (because `là` could be understood as `there`, beside `here`), let's translate `among us` directly.

```suggestion
msgstr "Ensemble, nous reconstruisons le web social. Nous sommes ravis que vous soyez parmi nous."
```